app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/actions/AuthenticatedActionWithRetrievedData.scala

```diff
@@ -170,9 +164,9 @@ class AuthenticatedActionWithRetrievedData @Inject()(
   private def hasSubscribed[A](enrolments: Enrolments, request: MessagesRequest[A])(
     implicit hc: HeaderCarrier
   ): Future[Either[Result, Option[CgtReference]]] =
-    enrolments.getEnrolment(CgtEnrolment.enrolmentKey) match {
+    enrolments.getEnrolment(EnrolmentConfig.Cgt.key) match {
```

Comment:
Import EnrolmentConfig so it just reads

```scala
enrolments.getEnrolment(Cgt.key)
```

Reply:
I'll probably rename it so that it reads `CgtEnrolment.key` - `Cgt.key` seems a bit ambiguous as to what it's referring to